libusb/os/linux_usbfs.h

```diff
@@ -164,10 +164,12 @@ int linux_udev_start_event_monitor(void);
 int linux_udev_stop_event_monitor(void);
 int linux_udev_scan_devices(struct libusb_context *ctx);
 void linux_udev_hotplug_poll(void);
-#else
+#elif !defined(__ANDROID__)
 int linux_netlink_start_event_monitor(void);
 int linux_netlink_stop_event_monitor(void);
 void linux_netlink_hotplug_poll(void);
+#elif defined(__ANDROID__)
+void android_jni_hotplug_poll();
```

Comment:
I have an error here with NDK 25.2.9519653.

```shell
error: this function declaration is not a prototype
void android_jni_hotplug_poll();
                             ^
                              void
```

Should here be added 'void' inside brackets?

```suggestion
void android_jni_hotplug_poll(void);
```